osf/models/mixins.py

```diff
@@ -26,14 +26,23 @@
     InvalidTagError,
     BlacklistedEmailError
 )
-from osf.models.node_relation import NodeRelation
-from osf.models.nodelog import NodeLog
-from osf.models.subject import Subject
-from osf.models.spam import SpamMixin, SpamStatus
-from osf.models.validators import validate_title
-from osf.models.tag import Tag
+from osf.models import (
+    NodeRelation,
```

Comment:
`ImportError: cannot import name 'NodeRelation'`

There _might_ be a reason that `NodeRelation` isn't in `osf.models.__init__`, but it could just as easily be an oversight